doc/sphinx/scripting/python/fontforge.rst

```diff
@@ -444,10 +444,23 @@ Module functions
 
 .. function:: open(filename[, flags])
 
-   Opens a filename and returns the font it contains (if any). If the flags
-   argument is 4, then ff will load all glyphs in the 'glyf' table of a ttc file
-   (rather than just the glyphs used in the font picked). This will not load all
-   'glyf' tables though.
+   Opens a filename and returns the font it contains (if any). The optional ``flags`` argument can be string tuple or integer combination of the following flags:
```

Comment:
Pretty sure this is wrong - in that the integer value is not accepted, only the string representation is (and if I'm right I hope to keep it that way, otherwise it just seems bad to expose the underlying representation to the user as an option)

Reply:
I'm going to assume you're reviewing the docs before the code content here ...

Probably because it is one of the older methods the `open` interface *only* has the integer flags option in the current `master`. This PR adds the string tuple. I'm not happy about the integer option either but @frank-trampe strongly encourages backwards compatible interface changes when possible.

Reply:
Sorry, yep you're right.